src/subcommand/wallet/runics.rs

```diff
@@ -0,0 +1,37 @@
+use super::*;
+
+#[derive(Serialize, Deserialize)]
+pub struct RunicUtxo {
+  pub output: OutPoint,
+  pub rune: SpacedRune,
+  pub rune_balance: u128,
+  pub amount: u64,
```

Comment:
```suggestion
```

I know I said to add this but after thinking about it I think we should just show the runes balance and not the btc amount.